db/migrate/20170713144355_create_dialog_field_associations_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreateDialogFieldAssociationsTable < ActiveRecord::Migration[5.0]
+  def change
+    create_table :dialog_field_associations, :id => false do |t|
+      t.integer "dialog_field_trigger_id", :null => false
+      t.integer "dialog_field_respond_id", :null => false
```

Comment:
- Please use symbols for consistency.
- Please remove the :null => false.
- All tables must have an id for replication purposes, so remove the :id => false